.github/workflows/nightly-test-integ-peering_commontopo.yml

```diff
@@ -106,12 +106,12 @@ jobs:
           TERM: ansi
 
       - name: Authenticate to Vault
-        if: ${{ endsWith(github.repository, '-enterprise') && !cancelled() }}
+        if: ${{ !cancelled() && endsWith(github.repository, '-enterprise') }}
         id: vault-auth
         run: vault-auth
 
       - name: Fetch Secrets
-        if: ${{ endsWith(github.repository, '-enterprise') && !cancelled() }}
+        if: ${{ !cancelled() && endsWith(github.repository, '-enterprise') }}
```

Comment:
This test suite was fixed previously in https://github.com/hashicorp/consul/pull/19783; noticed after most of the change was done so I just aligned order here for consistency.